### DISCUSSION TITLE
Roadmap

### DISCUSSION THREAD

**Discussion by PhillypHenning** [bitovi/bitops#245] [Community]
Eventually we'll need a roadmap to communicate to the users the project plans and direction.

This could be tied into 2 categories: 
# 1 Strategic Roadmap (where we want to go)
A nice looking landing page showcasing a very high level direction of the project.
* [Best example](https://images.squarespace-cdn.com/content/v1/5a168baaa8b2b09a2e8b0214/1626951925045-A2FDZE5KSS6BWUBFFG7E/roadmap.png)
* [Lots of content example](https://res.cloudinary.com/lmn/image/upload/c_limit,h_360,w_640/e_sharpen:100/f_auto,fl_lossy,q_auto/v1/gameskinnyc/g/r/e/green-hell-roadmap-b3b06.jpg)
* [High production value example](https://fyre.cdn.sewest.net/trm/6052904adcd4c20012874a02/roadmap_v17--2uLnX3Cu.png?width=&height=)

# 2 Tactical Roadmap (how to get there)
The other part is a list of major features and functionality. Something developers are expecting to find in the docs, that are planned for the future releases as well as what was done in the previous major releases. This could be as small as 2-3 bullet point per release highlighting the biggest changes, including the breaking ones. This is different from the very detailed CHANGELOG (#222) that's composed and maintained for each PR.
* [Roadmap/Backlog](https://docs.stackstorm.com/roadmap.html#backlog)
* [kubectl Roadmap](https://github.com/kubernetes/community/blob/master/sig-cli/migrated-from-wiki/roadmap-kubectl.md) composed by SIGs groups
* [Major Release History](https://docs.stackstorm.com/roadmap.html#release-history)
* [Changelog](https://github.com/StackStorm/st2/blob/master/CHANGELOG.rst) (just for reference)

----

And so we can share the BitOps strategy (where we want to go) on the BitOps website/landing page, Tactics (how to get there and what was done) in the docs Roadmap and provide more granular log of specific code changes per PR in the Changelog (#222).

**Comment by Pavelkoen** [comment]
Let's discuss it! 

belogupavel@gmail.com

**Reply by mickmcgrath13** [reply]
Thanks for your interest!  I've sent you an email.  Looking forward to discussing further.

**Comment by arm4b** [comment]
Following the https://github.com/bitovi/bitops/pull/277#discussion_r943820080, one of the topics to discuss perhaps during the next community meeting (TODO add to the agenda) (https://github.com/bitovi/bitops/pull/257) and also relevant to the Roadmap in general is how do we want to handle the deprecations, deprecation warnings, do we want to keep the functionality backward-compatible or not and if yes then for how many releases.

**Comment by mickmcgrath13** [comment]
# Need to decide:
- High level roadmap
- specific features

Maybe move the conversation into the first community call.

some high level notes:
- project quality
- project stability
- polished documentation
- improve landing page
- overview video (bitops 101)
- private plugin repos
- ops repo composability

# Next steps
- discuss during the first open community meeting
